jetty9/src/main/java/com/linecorp/armeria/server/jetty/JettyService.java

```diff
@@ -318,6 +318,9 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
                 });
                 success = true;
                 return null;
+            } catch (Exception e) {
+                res.abort(e);
+                throw e;
             } finally {
```

Comment:
Can we get rid of finally block (including the `success` flag) now that we catch an exception explicitly?  It seems to be a no-op now.

Reply:
Maybe should we catch `Throwable` then?